README.md

```diff
@@ -12,6 +12,10 @@ using a callback function in your controller.
 
 ![Autocomplete](http://i.imgur.com/wzGgfiY.gif)
 
+## Thanks!
```

Comment:
@earayo Could you remove this section of the readme pls? We will add an `Authors` sections later. Thank you.